.github/workflows/build_develop.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       repo: erpnext
       version: develop
       push: ${{ github.repository == 'frappe/frappe_docker' && github.event_name != 'pull_request' }}
+      platforms: ${{ github.event_name != 'pull_request' && 'linux/amd64,linux/arm64' || 'linux/amd64' }}
```

Comment:
This platforms will only affect Push step, so you can remove the condition here and allow both `linux/amd64` and `linux/arm64`.

Reply:
done